spec/models/question_spec.rb

```diff
@@ -50,4 +50,22 @@
       expect(subject.response_choices).to include(subject.question_orders.first.response_choice)
     end
   end
+
+  context 'question renders as sms' do
+    subject { FactoryGirl.create(:question, :with_multiple_response_choices) }
+    expectedString = "hello world?\na) first choice\nb) second choice"
```

Comment:
I think this is subjective but I usually use let for rspec definitions (i.e. `let(:expected_string) { "hello world" }`. I think it has to do with lazy loading and readability.